src/Agent.Worker/Handlers/HandlerFactory.cs

```diff
@@ -117,5 +127,31 @@ public sealed class HandlerFactory : AgentService, IHandlerFactory
             handler.TaskDirectory = taskDirectory;
             return handler;
         }
+
+        /// <summary> 
+        /// This method provides a list of in-the-box pipeline tasks for which we don't want to display the warning about the Node6 execution handler. 
+        /// </summary>
+        /// <remarks>We need to remove this method - once Node 6 handler is dropped</remarks>
+        /// <returns> List of tasks ID </returns>
+        private List<Guid> getTaskExceptionList()
+        {
+            var exceptionListFile = HostContext.GetConfigFile(WellKnownConfigFile.TaskExceptionList);
+            var exceptionList = new List<Guid>();
+
+            if (File.Exists(exceptionListFile))
+            {
+                try
+                {
+                    exceptionList = IOUtil.LoadObject<List<Guid>>(exceptionListFile);
+                }
+                catch (Exception ex)
+                {
+                    Trace.Info($"Unable to serialize exception list {ex}");
```

Comment:
Nit: this looks more like deserialization 😃

Reply:
Nice catch, fixed it.